DNN Platform/Library/Entities/Users/UserRegistrationEmailNotifier.cs

```diff
@@ -8,15 +9,17 @@ namespace DotNetNuke.Entities.Users
 {
     public class UserRegistrationEmailNotifier
     {
+        private static Lazy<UserInfo> CurrentUser => new Lazy<UserInfo>(() => UserController.Instance.GetCurrentUserInfo());
```

Comment:
Since this is a `static Lazy<T>`, it will only resolve once for the entire life of the application.  `UserController.Instance.GetCurrentUserInfo` is properly optimized enough that I don't think there's any problem adding another call to it in `NotifyAdministrator` if necessary.

Reply:
Fixed. Agree with you.